simulate_groups.py

```diff
@@ -76,7 +80,10 @@ def simulate_ll(n,
         num_groups = len(groups)
         if group_sparsity < 1.0:
             num_groups_to_keep = int(num_groups * group_sparsity)
-            groups_to_keep = np.random.choice(num_groups, num_groups_to_keep)
+            if prev_groups is None:
+                groups_to_keep = np.random.choice(num_groups, num_groups_to_keep, replace=False)
+            else:
+                groups_to_keep = np.random.choice(prev_groups, num_groups_to_keep, replace=False)
```

Comment:
Re. [this](https://github.com/greenelab/simulate-groups/pull/2/files#r656501484), I tested the case when the second run has a higher sparsity than the first run, and it throws an error. So I think what we want to do is this:
```python
num_groups = len(groups)
if group_sparsity < 1.0:
    if prev_groups is None:
        num_groups_to_keep = int(num_groups * group_sparsity)
        groups_to_keep = np.random.choice(num_groups, num_groups_to_keep, replace=False)
    else:
        num_groups_to_keep = int(prev_groups * group_sparsity)
        groups_to_keep = np.random.choice(prev_groups, num_groups_to_keep, replace=False)
```
Either you can make this change and regenerate the data, or I can make this change in a later PR (if this would make you have to rerun wenda again, I'm happy to stick with what we have for now and create an issue). Up to you.